js/game.js

```diff
@@ -922,6 +1064,13 @@ const entities = {
       friction: 1,
       restitution: 0.4,
     },
+    pineapple: {
+      shape: "circle",
+      radius: 20,
```

Comment:
This is proportionally too small. A pineapple should be bigger than an apple or orange.